lib/msf/core/modules/external/bridge.rb

```diff
@@ -212,6 +225,15 @@ def initialize(module_path, framework: nil)
     self.env = self.env.merge({'GOPATH' => go_path})
     self.cmd = ['go', 'run', self.path]
   end
+
+  def handle_exception(error)
+    case error
+    when Errno::ENOENT
+      LoadError.new('Failed to execute external Go module. Please ensure you have Go installed on your environment.')
+    else
+      error
```

Comment:
It probably makes sense to delegate to the `super` implementation here in the default scenario, same as the python bridge 👀

Reply:
Got his added in 👍